### ISSUE TITLE
[FE] - Auth Profile - Personal Info Revision - grep logs to find pages that link to /profile/

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#38292]
## Background

We will need to review content pages that currently link to /profile/ or more explicitly to /profile/personal-information/ on VA.gov.  We will need to see the context of the link to determine if they need to be redirected to the new /profile/contact-information page.

## Tasks
- [x] GREP logs to find pages that currently point to /profile/ or /profile/personal-information
- [ ] Share list of pages with team so they can look at context for redirect requests.

## Acceptance Criteria
- [x] grep logs to find pages that currently point to /profile/ or /profile/personal-information
- [x] Share list of pages with team so they can look at context for redirect requests.

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by MarciMcGuireGCIO** [comment]
@adamwhitlock1 I used to grep Apache logs 18 years ago before we had a tool for analytics.  It was so, so sad.

**Comment by adamwhitlock1** [comment]
@SKasimow and @Samara-Strauss I have compiled a list of apps and places in the code where the profile is linked.

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/personal-information/personal-information-revision/frontend/cross-app-profile-links.md

I think that should cover all our bases. I also tried to scrub through content build and make sure that there was no static content that linked to the profile, and I didn't find any references, but that content is a bit more opaque to search. I basically ran the content-build and then searched through the `build` folder for links.
